recording/src/nextcloud/talk/recording/Participant.py

```diff
@@ -33,8 +34,14 @@
 from secrets import token_urlsafe
 from selenium import webdriver
 from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.options import Options as ChromeOptions
+from selenium.webdriver.chrome.service import Service as ChromeService
+from selenium.webdriver.chrome.webdriver import WebDriver as ChromeDriver
+from selenium.webdriver.firefox.options import Options as FirefoxOptions
 from selenium.webdriver.firefox.service import Service as FirefoxService
+from selenium.webdriver.firefox.webdriver import WebDriver as FirefoxDriver
 from selenium.webdriver.support.wait import WebDriverWait
+
```

Comment:
Please remove the blank line to keep all `from library` imports together (although maybe they should be split between base libraries and external libraries... but just removing the line is simpler ;-) ).